### ISSUE TITLE
Deprecated dependencies (Next)

### ISSUE THREAD

**Issue by perlohmann** [keystonejs/keystone#5251]
Did a clean npm install and got several deprecated warnings on libraries that is no longer maintained and has been moved.
This was the warnings I got:

``` bash
npm WARN deprecated accept@3.1.3: This module has moved and is now available at @hapi/accept. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
```

In order to reproduce simply do a clean npm install with the following dependencies:
``` json
"dependencies": {
    "@keystone-next/admin-ui": "^12.0.0",
    "@keystone-next/auth": "^19.0.0",
    "@keystone-next/fields": "^5.3.0",
    "@keystone-next/keystone": "^14.0.0",
    "dotenv": "^8.2.0",
    "next": "^10.0.9",
    "react": "^17.0.1",
    "react-dom": "^17.0.1"
  },
```

OS: Win10

**Comment by JedWatson** [comment]
**tl;dr** this will be fixed soon!

@perlohmann thanks for the report. This is on our radar; those older versions are being included because Keystone 5 was compatible with node v10, and other than these warnings there has been no issue with that which would force users of those packages who are still on older versions of node to upgrade to continue getting releases.

tbh it's been a tricky situation because those packages were deprecated but their newer versions don't work with node v10, and when this happened Node v10 was still LTS. The LTS changed late last year by which point we were full steam ahead on the keystone-next packages, but in the same repo as keystone-5, so we couldn't bump the node version CI without errors.

anyway the good news is we're nearly done cleaning up the major transition we've been through, at which point the unused dependencies will all be gone from the repo and the `keystone-next` install, and the remainder will be able to be upgraded as Node v14 is now LTS and we're going to be targeting that as the lowest supported version going forward (our goal is to always support the LTS releases)

looking forward to putting this behind us 🙂

**Comment by timleslie** [comment]
A clean install of `keystone-next/@keystone` results in the following deprecation warnings:

```
npm WARN deprecated accept@3.1.3: This module has moved and is now available at @hapi/accept. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
```

These stem from the `apollo-server-micro` package, which is currently maintained at its most recent version. As such, there is no further action for us to take on this issue other than maintaining our regular updates of upstream packages 👍
